example/flutter/README.md

```diff
@@ -0,0 +1,16 @@
+# cronet_example
```

Comment:
Replace this boilerplate readme with a description of what the Flutter example is doing.

Reply:
I'll delete the cli example as we're going flutter only for now :). So, I'll delete this readme file also and pub will index the actual code under `Example` tab by it's own. :)

Reply:
Why only Flutter? The Dart CLI things just keep working if you have a `dart` on your path correct?

Reply:
When I try to do `dart run`, I get -
```
Resolving dependencies... 
Because example_dart depends on cronet from path which requires the Flutter
  SDK, version solving failed.

Flutter users should run `flutter pub get` instead of `pub get`.
```

Reply:
Oops. It does work.
If I do - `dart run bin/example_dart.dart` it works.
But, if I do `dart run`, it gives the error I mentioned above. Keep the CLI example then. :)